added bitscrunch as linea indexers (#966)
Consensys/doc.linea: docs/get-started/tooling/data-indexers/bitscrunch.mdx

---
title: bitsCrunch

> description: bitsCrunch is an AI-Enabled Decentralized Blockchain Data Network

Reviewer: ```suggestion
description: bitsCrunch is an AI-enabled decentralized blockchain data network
```

---

# bitsCrunch

> bitsCrunch is an AI-powered on-chain data indexer which offers 100+ endpoints for NFTs, wallets, tokens, DeFi and more, through a single API. Along with providing access to the most comprehensive on-chain data set, it also offers exclusive AI analytics like IP infringement detection and asset price estimation.

Reviewer: ```suggestion
bitsCrunch is an AI-powered onchain data indexer which offers 100+ endpoints for NFTs, wallets, tokens, DeFi and more, all through a single API. Along with providing access to the most comprehensive onchain data set, it also offers exclusive AI analytics like IP infringement detection and asset price estimation.
```

## Integrate bitsCrunch to get access to:

> 1. **NFT Data and Analytics** - NFT metadata, transaction history, wash trading detection and more

Reviewer: ```suggestion
1. **NFT data and analytics** - NFT metadata, transaction history, wash trading detection and more
```

> 2. **Token Data and Analytics** -  Token price, liquidity, reputation scores and more

Reviewer: ```suggestion
2. **Token data and analytics** -  Token price, liquidity, reputation scores and more
```

> 3. **Wallet Data and Analytics** - Wallet portfolio tracking, history, reputation scores and more

Reviewer: ```suggestion
3. **Wallet data and analytics** - Wallet portfolio tracking, history, reputation scores and more
```

> 4. **Gaming Data and Analytics** - Engagement rates, revenue tracking, active wallets and more

Reviewer: ```suggestion
4. **Gaming data and analytics** - Engagement rates, revenue tracking, active wallets and more
```

> 5. **DeFi Data and Analytics** - TVL, liquidity pools, positions tracking and more

Reviewer: ```suggestion
5. **DeFi data and analytics** - TVL, liquidity pools, positions tracking and more
```
